docs/developer-docs/production/system-canisters.mdx

```diff
@@ -0,0 +1,24 @@
+# System canisters
+
+## Overview
+
+System canisters are canisters that provide a necessary function to the ICP network. These canisters are controlled by the NNS and are upgraded via proposals. These canisters have static canister IDs that projects can reference. One common example of a system canister that is referenced by projects is the Internet Identity canister. 
+
+## System canisters
+
+| System canister | Canister ID |
+|-----------------|-------------|
+| NNS registry        | rwlgt-iiaaa-aaaaa-aaaaa-cai |
+| NNS governance      | rrkah-fqaaa-aaaaa-aaaaq-cai |
+| ICP index       | qhbym-qaaaa-aaaaa-aaafq-cai |
+| ICP ledger          | ryjl3-tyaaa-aaaaa-aaaba-cai |
+| ICP ledger archive | qsgjb-riaaa-aaaaa-aaaga-cai |
+| ICP ledger archive | qjdve-lqaaa-aaaaa-aaaeq-cai |
+| NNS root | r7inp-6aaaa-aaaaa-aaabq-cai |
+| Cycles minting | rkp4c-7iaaa-aaaaa-aaaca-cai |
+| NNS lifeline | rno2w-sqaaa-aaaaa-aaacq-cai |
+| Genesis token | renrk-eyaaa-aaaaa-aaada-cai |
+| SNS wasm | qaa6y-5yaaa-aaaaa-aaafa-cai |
+| Internet Identity | rdmx6-jaaaa-aaaaa-aaadq-cai |
+| NNS UI | qoctq-giaaa-aaaaa-aaaea-cai |
+| Motoko playground | m7sm4-2iaaa-aaaab-qabra-cai |
```

Comment:
This is debatable. Motoko playground is a canister developed by DFINITY but it's not controlled by NNS nor does it provide critical functionality (as other canisters listed here). It's certainly a useful canister but I think it's best to not include it in this list.

We could consider adding the bitcoin (mainnet: `ghsi2-tqaaa-aaaan-aaaca-cai` and testnet: `g4xu7-jiaaa-aaaan-aaaaq-cai`) canisters. I think those two fit the bill -- they provide functionality that extends the capabilities of ICP and are controlled by the NNS.

Reply:
We should also add the Exchange Rate Canister. I'm also on the fence about adding ckBTC and ckETH canisters here, or if this information should be only in the dedicated section of the docs.

Reply:
Removed Motoko playground and added the Bitcoin testnet/mainnet. For ckBTC/ckETH, I think they would fit best on the pages dedicated to each.